### PR TITLE
Data Requests Support

### PR DESCRIPTION
**Pull request by nixj14**
Changes per: https://docs.google.com/document/d/199ehx6fvsZ2uksgOy1sFq5VkhCJ8_i5QU47SMzRzFF0/edit

### CODE REVIEW

File: ui/src/main/webapp/picsureui/common/session.js [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+    };
+    $.when(queryTemplateRequest(), meRequest(), userRequest()).then(
+      function (queryTemplateResponse, meResponse, userResponse) {
+        var currentSession = JSON.parse(sessionStorage.getItem("session"));
+        currentSession.queryTemplate = queryTemplateResponse[0].queryTemplate;
+        // Check for data admin role from users
+        var meUUID = meResponse[0].uuid;
+        var meUserIndex = userResponse[0].map((u) => u.uuid).indexOf(meUUID);
+        var meUser = userResponse[0][meUserIndex];
+        if (meUser.roles.map((r) => r.name).indexOf("Data Admin") != -1) {
```
Comment (chunacatsunflower) [inline ui/src/main/webapp/picsureui/common/session.js]: This should be a privilege, created using a SQL script here: https://github.com/hms-dbmi/pic-sure-gic-institution/tree/main/custom-migrations/auth

File: custom-migrations/auth/V8__SITE_ADMIN.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+use auth;
+
+ SET @uuidRole = REPLACE(UUID(),'-','');
+  INSERT INTO role VALUES (
+      unhex(@uuidRole),
+     'Data Admin',
+     'Allow access to Dataset Requests tab'
+  );
```
Comment (chunacatsunflower) [inline custom-migrations/auth/V8__SITE_ADMIN.sql]: creating and linking a Privilege to this role will allow us to avoid overriding the session.js code

Comment (nixj14) [reply custom-migrations/auth/V8__SITE_ADMIN.sql]: ok.  didn't realize roles were not being used for this purpose.  Will create a privilege and have the developer update to remove unnecessary changes.

File: ui/src/main/webapp/picsureui/styles.css [hunk 3]
```diff
@@ -0,0 +1,841 @@
+#main-content {
```
Comment (chunacatsunflower) [inline ui/src/main/webapp/picsureui/styles.css:1]: We should not need to override this huge file.  Please add any styles needed for the new pages in the styles override file, https://github.com/hms-dbmi/pic-sure-gic-institution/blob/main/ui/src/main/webapp/picsureui/overrides/styles.css

**Pull request merged**